### ISSUE TITLE
Bug report: Application crashed with TraitError

### ISSUE THREAD

**Issue by cpignedoli** [aiidalab/aiidalab-qe#542]
## Automated report

_This issue was created with the app's automated bug reporting feature.
Attached to this issue is the full traceback as well as an environment
fingerprint that contains information about the operating system as well as all
installed libraries._

## Additional comments (optional):

not sure what happened I was selecting a structure from the AiiDA database

## Attachments

<details>
<summary>Traceback</summary>

```python-traceback
/opt/conda/lib/python3.9/site-packages/traitlets/traitlets.py in _notify_trait(self, name, old_value, new_value)
   1372 
   1373     def _notify_trait(self, name, old_value, new_value):
-> 1374         self.notify_change(
   1375             Bunch(
   1376                 name=name,

/opt/conda/lib/python3.9/site-packages/ipywidgets/widgets/widget.py in notify_change(self, change)
    692                 # Send new state to front-end
    693                 self.send_state(key=name)
--> 694         super(Widget, self).notify_change(change)
    695 
    696     def __repr__(self):

/opt/conda/lib/python3.9/site-packages/traitlets/traitlets.py in notify_change(self, change)
   1384     def notify_change(self, change):
   1385         """Notify observers of a change event"""
-> 1386         return self._notify_observers(change)
   1387 
   1388     def _notify_observers(self, event):

/opt/conda/lib/python3.9/site-packages/traitlets/traitlets.py in _notify_observers(self, event)
   1429                 c = getattr(self, c.name)
   1430 
-> 1431             c(event)
   1432 
   1433     def _add_notifiers(self, handler, name, type):

/opt/conda/lib/python3.9/site-packages/traitlets/traitlets.py in _update_source(self, change)
    379             setattr(self.source[0], self.source[1], self._transform_inv(change.new))
    380             if getattr(self.target[0], self.target[1]) != change.new:
--> 381                 raise TraitError(
    382                     "Broken link {}: the target value changed while updating "
    383                     "the source.".format(self)

TraitError: Broken link <traitlets.traitlets.link object at 0xffff3b403640>: the target value changed while updating the source.
```
</details>

<details>
<summary>Environment fingerprint</summary>
<pre>eNpdWF1v47gV_SuB-7oSLMl2JvO2HezLYtKmm2kXxWIRUBJtM6ZIRqTiKIv5771fVDR99CFF3o9zz730X5tXPUbj3eZz9dMmWJWO
fhw2n__aqLE7m6S7NI168_mPzWHXmrT5abP5E_bN6ezd0_LppinvyqqB1Q_ob9XN4_3DzcNvv_xy__Dt5tt5unnU4ebm9mZ7-3n3
6fPu9ubf377c1Nu62XyHI1V3UScd6WpjelV0Hi_e1GVTVnAygyHqqfcAVwBvF_hlUi5Ng45h1DHi-m69blWLJzX_hxXaoqXbsirr
NXz2g877q_XCi8DVtty2P6xcTX_SKRatimx0VW4V3-WLYC4KwEP5SY6DEL9Q1OR8q4fWdOSVbHHR1Oc0WMHoJDcbT1_d8p4QejNG
2rErd4wMvtfkknyDSXzVRVDpTPCOPR1P3tVFdzwaNHaJ5QdctMb1xp0ir9e8Tr41ZV2zATElf9GO9pRwJWGz64qTdnpUyY9kXEXf
pkS21hQaAP6uWm05VNsSqdMCAzplLdnJFyIU_JhieZxcl7y3sbDjVHSqO2s6-0COt904h0RZ35JprVZTMsfJRj-
FHS1UFa9YDR8Dss9brXEXzZbu8Te4dCa7dmzW6JM1YSazbsmsL3j9F-_S6NnaCpxAO8guMpOurEsGs-uNEK3TIxhHoQf2w6H1gso
9FV0t-QHD92Rqd1Zj1KlwwB9lzTtZjfHkRT0Yd_Rywh1Z2pmxmyIfyYzorOkuAADPytv8u4jBOEfH4d2UsM5bP6pBCW32BLlecQ7
leAQKKd3irFwPwTyRycuG5KeRbKqyoZgrfxpVOCO8AzpwnObOZhOYYL3qQIFWJdCrpDqrYtTsEzKu1-10kgsOFLZeg3QI-_Zyaa-
PU9T922AljwyCXEB6WLAyMXsTL4Vvn0H6YmIN2sLJtOQiS5_wA9PW-y6mEdyGOEB2xAEMVz_Zq-m47qCGDgBpqHTtkpRjXe7JYD0
oY4tXSGm_FA0rA-yFaHnjEjuMcdJvnQ5o82kEcktg6Zg33QHrJQHs9xGKVAXDRm135AViz9G7iJRR7PmBfDkaq60nfjSYBLzuaE1
6Ei3O0nOEtH7QfMe2HiduFcw1NAgUsVDO-UQhjiuppey_mneODRfiadTaWZ1WHp2nQbnjaDQQizK8JQdM7xRZg_aZARXCmrYYdFL
IEC53lrSPVWgMwMSOmLMv80lhhtp3JEQHjIKAQ7BSRQ0DkvTbsmnyHoRQ6SDkNq5EC5fAFUFqRqQ_0BG3bPcCFvotgYe6X-XNpNg
rd9KQ4ijcxJOedW9yhHHbr8Y9q5rTxRs86FkrDCIAEr0XX_bymwgl4rETbNRHtlh-L-RoxK3nCfzS4xMIBrCSY8En5gXmyKK0ghd
QE6-ir9XdDyuJygEYLsxYXXOeWrp7uwa5jUPaqfA_wCLMJyyruIr5x-rTYkCNQvrjYvGRmEoU8WKuH3KfgehtdoK10Krx8kPFc5R
ggIL6hCbcSVNEfyEjgxpaJSrFYo51hvWWv64JO03jJFcjcK8uPOzUREQ6ZXXGPRjR-ytPYHsBpvCojnp1_aBSsNjEOHospR9gYRy
otthBKjWYmCaXj0CGDFDwc-5Xd2W9J2zUBejzmJUAKMkCM4SBBS4PFkM8YZMgZMtXTDaZ3nSJCo8vcSpFKFY6iQ9yLak9jUa5CSE
mDMz9GCDvID3MSu4drsVBViWq9gNDOoIe4YBivATxQHC6-vHyxlxH4GRfjb4K_5BqIGG69f5CxgLB6xVWxLMZVgR20yB5rllOPIw
i5xx8Ns6Dfg-q1-vGnLFicQ-CjY22utvk8Zi1vWZxDDA2pqRaq2WUZHDQVkmgtwy8BBmj-
Te07CjuNwL4DpgImcw4b4Tub4h-dWY38t3LeEnf6regOx682ItwipNUIaY0wHBhdYS5RUu-MIUPxlp_ZT3noD0YSWjFqQom8MDED
A6XE6rsE2o41OGTU4POXZJuRSbPLOxMiWAlDdR6sYrDCFN9OuspFisCAWMrWQypQCZfTJLMN7QS8WY6-o7YEeLc-XCqKS53fLRAO
DQrUrNlJc1wNDSduKJrgDZZaGj3K7kKcwvCwQBz_2H-Yo6__c4dlsIW4JaokswjDSOLBME5FUPTaKkQdiwSYYZOkuR5gRFv6PRFM
evcKx_mX3__JmMJ2TlD_UCDy3WP31a0735-_NfXVYN8mP-hvtg1eeZ_Bu0eH79yFoU9MxrLHMb4Eq9nQGJa0sFt7WF-BHVkNt6KE
9Ruob1ryUcNjKs_VrBnFaCfJwnGNn_mnzs1hFWBYntWTA2eIAB4Zx8r8ea_P99_FWHiU97pvVbn58WLLrLo5alx1C8T6AtDNbuRs
eW9gvH6tF4YvKhiJaccu-bu061wm7ZOWNHlrGRyBX5Tnld42bG6IJcOy1KBS0Ue2XEkLT_hFbEzeRxnd6PWl5D1mmn7CONI_W1U8
bwavuHlMYXs9mFfcuXEM0j2NJleNIcYEs0b38AKGJ2Bl8z61Y7Psmj0a37g12XwMdGn4cTe5MAC0362OIzM-
aVL2-AhcClk2ttKOwRwRE2U2YurDYfzLhUY2aa54zJEeM4yTWoOOjph7xZxI1UBVuIDZKaGxA0YNBIeWar3Waxon3FzF2O9Gt-
Sz-niiQp_mxUvMYzv67kBxn45lifx9NIPVPoHDjNQ1iTLg8peuiNoCxTTE02PUQbsrMMT1gPtld8On5cKZNbQBNzIsDOFI1C5kN1
b0ThQkI9HpoS9YRyS0-TuRrn-j4IJe17N9leVunPvT6v_bq4djFnyD0RNWnbVLT0wZfBqGNIORif51yEnFeAIVQJPiUW2K-He9aw
1h7lhleBhrmihpUXsCetjaOkp6NF4nDuWvlmtlqNrl2jSIifjCq-
VlB_iGE14Ri7z8VJHsxqt1BB-9G5CkH_Gtpvv3_8HYop5AA==</pre>
</details>

**By submitting this issue I confirm that I am aware that this information can
potentially be used to determine what kind of calculation was performed at the
time of error.**

**Comment by unkcpz** [comment]
This should be fixed by https://github.com/aiidalab/aiidalab-qe/pull/537

**Comment by unkcpz** [comment]
Fixed by #537
